flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
arm64 is available https://data.trezor.io/suite/releases/desktop/latest/Trezor-Suite-24.10.1-linux-arm64.AppImage

Reply:
Yes, I wanted to focus on ARM64 later, as I wanted to check the viability of this project first. Is it possible to add it later, or would you require both architectures at the same time?

Reply:
I prefer that you have the arm64 build started. It's just one extra source. Without a build no one will be able to test it on arm machines.